source/read/specify-a-query.txt

```diff
@@ -0,0 +1,265 @@
+.. _kotlin-sync-specify-query:
+
+===============
+Specify a Query
+===============
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+.. facet::
+   :name: genre
+   :values: reference
+
+.. meta::
+   :keywords: expressions, operations, read, write, filter
+
+Overview
+--------
+
+In this guide, you can learn how to specify a query by using the {+driver-short+}.
+
+You can refine the set of documents that a query returns by creating a
+**query filter**. A query filter is an expression that specifies the search
+criteria MongoDB uses to match documents in a read or write operation.
+In a query filter, you can prompt the driver to search for documents with an
+exact match to your query, or you can compose query filters to express more
+complex matching criteria.
+
+Sample Data
+~~~~~~~~~~~
+
+The examples in this guide run operations on a collection called
+``fruits`` that contains the following documents:
+
+.. code-block:: json
+
+   { "_id": 1, "name": "apples", "quantity": 5, "rating": 3, "color": "red", "type": ["fuji", "honeycrisp"] },
+   { "_id": 2, "name": "bananas", "quantity": 7, "rating": 4, "color": "yellow", "type": ["cavendish"] },
+   { "_id": 3, "name": "oranges", "quantity": 6, "rating": 2, "type": ["naval", "mandarin"] },
+   { "_id": 4, "name": "pineapple", "quantity": 3, "rating": 5, "color": "yellow" },
+
+The documents in this collection are modeled by the following {+language+} data class:
+
+.. literalinclude:: /includes/read/specify-a-query.kt
+   :start-after: start-data-class
+   :end-before: end-data-class
+   :language: kotlin
+   :copyable:
+
+The following code example shows how to create a database and collection, then
+insert the sample documents into your collection:
+
+.. literalinclude:: /includes/read/specify-a-query.kt
+   :start-after: start-sample-data
+   :end-before: end-sample-data
+   :language: kotlin
+   :dedent:
+   :copyable:
+
+Exact Match
+-----------
+
+Literal value queries return documents with an exact match to your query filter.
+
+The following example specifies a query filter as a parameter to the ``find()``
+method. The code returns all documents with a ``color`` field value of ``"yellow"``:
```

Comment:
Our guidance isn't 100% clear on what "introducing" means, but the style [guide on colons](https://www.mongodb.com/docs/meta/style-guide/style/punctuation/#colons) says: "If another sentence intervenes between the introduction and the thing being introduced, use a period instead of a colon." I think that might apply to these examples? Since we say "The following example..." in the first sentence, but then have a clarifying sentence in between. 
I can see a case where both sentences are introduction sentences though, so will differ to your judgement on whether or not we change these.

Applies to most sections

Reply:
I agree with your interpretation of the style guide here. Can change these to periods.